docs/build/build-free-loop.md

```diff
@@ -119,8 +110,9 @@ Some features of Loop are not available with the Free option, so as you sign, yo
     * **Watch App Extension Target:** Siri
 1. Add the keyword `SIRI_DISABLED` to the LoopConfigOverride.xcconfig file
     * Click on the filename in the left pane of Xcode and view it in the Xcode editor
-    * Examine the file and find the line that starts with `SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(inherited)`
+    * Examine the file and find the line that starts with<br> `SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(inherited)`
```

Comment:
From the top of my head, suffixing the line with **2 spaces** should also do the trick and add a new line.